tests/reference/ast_new-string2-44323ea.stdout

```diff
@@ -1,12 +1 @@
-(Module [(Expr (ConstantStr "\"\\" ())) (Expr (ConstantStr "
-\\" ())) (Expr (ConstantStr "\\" ())) (Expr (ConstantStr "   \\" ())) (Expr (ConstantStr "\\ " ())) (Expr (ConstantStr "\
-" ())) (Expr (ConstantStr "\\|" ())) (Expr (ConstantStr "Text\\" ())) (Expr (ConstantStr "\
-" ())) (Expr (Call (Name x Load) [(ConstantStr "\
-class a:
-    def test():
-        return r"\\LaTeX"
-" ())] [])) (Expr (ConstantStr "
-    "\\"
-" ())) (Expr (ConstantStr "
-    '\'
-" ()))] [])
+(Module [(Expr (ConstantStr ""\\" ())) (Expr (ConstantStr "\n\\" ())) (Expr (ConstantStr "\\" ())) (Expr (ConstantStr "   \\" ())) (Expr (ConstantStr "\\ " ())) (Expr (ConstantStr "" ())) (Expr (ConstantStr "\\|" ())) (Expr (ConstantStr "I said 'Wow' " ())) (Expr (ConstantStr "I said "hi" " ())) (Expr (ConstantStr "I said \\"hi" " ())) (Expr (ConstantStr "I said '\\'hi\\hey' " ())) (Expr (ConstantStr "ab" ())) (Expr (ConstantStr "ab" ())) (Expr (ConstantStr "Text\\\\" ())) (Expr (ConstantStr "" ())) (Expr (Call (Name x Load) [(ConstantStr "class a:\n    def test():\n        return r"\\LaTeX"\n" ())] [])) (Expr (ConstantStr "\n    "\\\\"\n" ())) (Expr (ConstantStr "\n    '\\'\n" ()))] [])
```

Comment:
Everything seems to be correct. Except `""\\"`, which should be `'"\\'` or `"\"\\"`. I think we don't unescape `"` in this PR. Instead, Let's create a new issue or work on another PR.

Reply:
Yes, please create a new issue for this for a separate discussion.
Right now what I think is, in the ```ConstantStr``` node we use ```string value``` while python use ```constant value```. I think this is why our strings are wrapped in double quotes instead of single quotes, which is the only difference I see here.

Reply:
Yes, please create a new issue to work on this.
Right now what i think is, in Lpython's ```ConstantStr``` node we use ```string value``` while python uses ```constant value```. I think this is why the strings are wrapped in double quotes instead of single quotes, which is the only difference I see here. If fixing this is simply not unescaping ```"```, that will be easy and I'll do it in my next commit in this PR only.